.github/workflows/pull-request-checks.yml

```diff
@@ -136,9 +136,18 @@ jobs:
           crate: mdbook-cmdrun
           version: latest
 
+      - name: Cargo build tedge cli
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: -p tedge
+
       - name: Run mdbook build
+        with:
+          path: target/debug
         run: |
           ls -l
+          echo $PATH
```

Comment:
Is printing PATH here intentional?

Reply:
Yes - this is a draft PR aimed to check the behavior of mdbook-cmdrun.